core/txpool/legacypool/legacypool.go

```diff
@@ -1442,7 +1442,7 @@ func (pool *LegacyPool) promoteExecutables(accounts []common.Address) []*types.T
 		}
 		log.Trace("Removed old queued transactions", "count", len(forwards))
 		// Drop all transactions that are too costly (low balance or out of gas)
-		drops, _ := list.Filter(pool.currentState.GetBalance(addr), gasLimit)
+		drops, _ := list.Filter(pool.currentState.GetBalance(addr).ToBig(), gasLimit)
 		for _, tx := range drops {
```

Comment:
I guess for next step, we should change the big.Int in txpool to uint256.Int as well right? Just not in this PR.

Reply:
yep. IIRC there's already a PR to that effect